cmd/format-xl.go

```diff
@@ -124,6 +124,25 @@ type formatXLV3 struct {
 	} `json:"xl"`
 }
 
+func (f *formatXLV3) Clone() *formatXLV3 {
```

Comment:
If it can be avoided I generally would like to avoid stuff that requires access to all fields since it makes it easy to forget stuff when adding new fields, etc.

Since the struct is assumed to be able to do a JSON roundtrip, this should be easier to maintain:

```Go
func (f *formatXLV3) Clone() *formatXLV3 {
	b, err := json.Marshal(f)
	if err != nil {
		panic(err)
	}
	var dst formatXLV3
	err = json.Unmarshal(b, &dst)
	if err != nil {
		panic(err)
	}
	return &dst
}
```
Since this isn't a critical function I think the byte slice allocation is reasonable.

Reply:
Yes this will work too @klauspost